File: Euclid/Game/Room/Util/RoomUserStatus.cs
namespace Euclid.Game
{
    public class RoomUserStatus
    {
        #region Fields
        #endregion

        #region Properties

        public string Key { get;  }
        public string Value { get; }

        #endregion

        #region Constructor

        public RoomUserStatus(string key, string value)
        {
            Key = key;
            Value = value;
        }

        #endregion
    }
}

File: euclid/src/main/java/euclid/game/room/RoomUserStatus.java
package euclid.game.room;

public class RoomUserStatus {

    private final String key;
    private final String value;

    public RoomUserStatus(String key, String value) {
        this.key = key;
        this.value = value;
    }

    public String getKey() {
        return key;
    }

    public String getValue() {
        return value;
    }
}
